src/client/game/game-reducer.js

```diff
@@ -1,15 +1,15 @@
-
 import * as actionTypes from './game-actions';
 
 const initialState = {
 	started: false,
-	leader: {
+    leader: {
         id: undefined, 
```

Comment:
If you want to keep these, maybe go for null instead unless you're checking explicitly for undefined?